lib/api3/swagger.js

```diff
@@ -10,7 +10,7 @@ function setupSwaggerUI (app) {
   const serveSwaggerDef = function serveSwaggerDef (req, res) {
     res.sendFile(__dirname + '/swagger.yaml');
   };
-  app.get('/swagger.yaml', serveSwaggerDef);
+  app.get('/swagger', serveSwaggerDef);
```

Comment:
This is a breaking change which will result in a 404 for users who have bookmarked this page or scripts that programmatically request this route.

Do we need to change this? If so, what do you think about supporting both `/swagger.yaml` and `/swagger`?routes?

Reply:
You are right that it looks like `/swagger.yaml` URL has changed, but in fact it works both `/swagger.yaml`, `/swagger`. 
Actually `/swagger.xxx` returns JSON everytime, no matter which file extension you use.
Only READ, SEARCH, HISTORY operations (GETs for some collection) evaluate the `Accept`/extension, so it should be fine.